kobo/apps/superuser_stats/views.py

```diff
@@ -86,18 +325,22 @@ def user_details_report(request):
     )
     generate_user_details_report.delay(filename)
```

Comment:
This doesn't appear to be working correctly either as I'm only getting `AnonymousUser` included in the export.

Reply:
Fixed!